File: Coder Revolt - Web Browser Open Source/Coder Revolt - Web Browser Open Source/PageSource.vb
Public Class PageSource
    Private Sub PageSource_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.TopMost = True
        Me.txtURL.Text = Main.ComboBox1.Text

        btnViewSource.PerformClick()

    End Sub

    Private Sub btnViewSource_Click(sender As System.Object, e As System.EventArgs) Handles btnViewSource.Click
        btnViewSource.Enabled = False


        Dim WebStream As System.Net.WebClient
        Dim SourceCodeStream As System.IO.Stream
        WebStream = New System.Net.WebClient
        Try
            SourceCodeStream = WebStream.OpenRead(txtURL.Text)
            Dim objReader As System.IO.StreamReader = New System.IO.StreamReader(SourceCodeStream)
            txtSource.Text = objReader.ReadToEnd
        Catch
            MsgBox("Invalid URL" & vbCrLf & "File looking for cannot be found.", MsgBoxStyle.Exclamation, "Invalid URL")
        End Try

        btnViewSource.Enabled = True
    End Sub
End Class

File: C# Converted Code V.1/PageSource.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class PageSource
{
	private void PageSource_Load(System.Object sender, System.EventArgs e)
	{
		this.TopMost = true;
		this.txtURL.Text = Main.ComboBox1.Text;

		btnViewSource.PerformClick();

	}

	private void btnViewSource_Click(System.Object sender, System.EventArgs e)
	{
		btnViewSource.Enabled = false;


		System.Net.WebClient WebStream = default(System.Net.WebClient);
		System.IO.Stream SourceCodeStream = default(System.IO.Stream);
		WebStream = new System.Net.WebClient();
		try {
			SourceCodeStream = WebStream.OpenRead(txtURL.Text);
			System.IO.StreamReader objReader = new System.IO.StreamReader(SourceCodeStream);
			txtSource.Text = objReader.ReadToEnd;
		} catch {
			Interaction.MsgBox("Invalid URL" + Constants.vbCrLf + "File looking for cannot be found.", MsgBoxStyle.Exclamation, "Invalid URL");
		}

		btnViewSource.Enabled = true;
	}
	public PageSource()
	{
		Load += PageSource_Load;
	}
}